astroid/interpreter/_import/spec.py

```diff
@@ -157,6 +157,19 @@ def find_module(
                         location=getattr(spec.loader_state, "filename", None),
                         type=ModuleType.PY_FROZEN,
                     )
+                if (
```

Comment:
Could this clash with `namespace` modules?

Reply:
Could you flesh this out a little more for me? (Sorry!)

Reply:
```console
❯ tree .
.
├── argparse.py
└── test.py

1 directory, 2 files
```

```python
# test.py

import argparse

argparse.i_dont_exist()
```

```python
# argparse.py

def i_dont_exist():
    print("1")
```

```console
❯ python test.py
1
```

Won't this implementation default to `stdlib.argparse` instead of the local `argparse`?

Reply:
Nope, set up this example and it's looking good 😎 

I added a call to argparse.ArgumentParser to make sure it's unavailable:

```
************* Module test
test.py:8:0: C0305: Trailing newlines (trailing-newlines)
test.py:1:0: C0114: Missing module docstring (missing-module-docstring)
test.py:6:0: E1101: Module 'argparse' has no 'ArgumentParser' member (no-member)

------------------------------------------------------------------
Your code has been rated at 0.00/10 (previous run: 0.00/10, +0.00)
```

Reply:
Oops, I didn't run the test case against the correct copy of astroid. More soon.